### PR TITLE
Fix kubectl patch command for HelmRelease and HelmRepository resources

### PR DESCRIPTION
**Pull request by JakeSCahill**
Fixes https://github.com/redpanda-data/documentation-private/issues/2199

### CODE REVIEW

File: modules/upgrade/pages/migrate/kubernetes/helm-to-operator.adoc [hunk 1]
```diff
@@ -235,8 +169,8 @@ kubectl edit pod <operator-name> --namespace <namespace>
 ```bash
 kubectl --namespace <namespace> patch redpanda <cluster-name> -p '{"metadata":{"finalizers":null}}' --type=merge
 kubectl --namespace <namespace> patch helmrelease <cluster-name> -p '{"metadata":{"finalizers":null}}' --type=merge
-kubectl --namespace <namespace> patch helmchart <cluster-name> -p '{"metadata":{"finalizers":null}}' --type=merge
-kubectl --namespace <namespace> patch helmrepository <cluster-name> -p '{"metadata":{"finalizers":null}}' --type=merge
+kubectl --namespace <namespace> patch <namespace>-helmchart <cluster-name> -p '{"metadata":{"finalizers":null}}' --type=merge
+kubectl --namespace <namespace> patch <namespace>-helmrepository <cluster-name> -p '{"metadata":{"finalizers":null}}' --type=merge
```
Comment (RafalKorepta) [inline modules/upgrade/pages/migrate/kubernetes/helm-to-operator.adoc]:
```suggestion
kubectl --namespace <namespace> patch helmrepository <helm-repository-name> -p '{"metadata":{"finalizers":null}}' --type=merge
```

## Reference

https://github.com/redpanda-data/redpanda-operator/blob/fa12e1139a687ae49e7c1e089c719c7d9c28eb1b/src/go/k8s/api/redpanda/v1alpha1/redpanda_types.go#L171-L177

File: modules/upgrade/pages/migrate/kubernetes/helm-to-operator.adoc [hunk 2]
```diff
@@ -235,8 +169,8 @@ kubectl edit pod <operator-name> --namespace <namespace>
 ```bash
 kubectl --namespace <namespace> patch redpanda <cluster-name> -p '{"metadata":{"finalizers":null}}' --type=merge
 kubectl --namespace <namespace> patch helmrelease <cluster-name> -p '{"metadata":{"finalizers":null}}' --type=merge
-kubectl --namespace <namespace> patch helmchart <cluster-name> -p '{"metadata":{"finalizers":null}}' --type=merge
-kubectl --namespace <namespace> patch helmrepository <cluster-name> -p '{"metadata":{"finalizers":null}}' --type=merge
+kubectl --namespace <namespace> patch <namespace>-helmchart <cluster-name> -p '{"metadata":{"finalizers":null}}' --type=merge
```
Comment (RafalKorepta) [inline modules/upgrade/pages/migrate/kubernetes/helm-to-operator.adoc]:
```suggestion
kubectl --namespace <namespace> patch helmchart <namespace>-<cluster-name> -p '{"metadata":{"finalizers":null}}' --type=merge
```

## Reference

https://github.com/fluxcd/helm-controller/blob/610f6e6d38cd2c0e3b75dc6a403cc0541146c48a/api/v2beta1/helmrelease_types.go#L1149-L1152
and
https://github.com/fluxcd/helm-controller/blob/610f6e6d38cd2c0e3b75dc6a403cc0541146c48a/internal/reconcile/helmchart_template.go#L205
and
https://github.com/redpanda-data/redpanda-operator/blob/fa12e1139a687ae49e7c1e089c719c7d9c28eb1b/src/go/k8s/internal/controller/redpanda/redpanda_controller.go#L768
and
https://github.com/redpanda-data/redpanda-operator/blob/fa12e1139a687ae49e7c1e089c719c7d9c28eb1b/src/go/k8s/api/redpanda/v1alpha1/redpanda_types.go#L167-L169

**Review by RafalKorepta — CHANGES_REQUESTED** [review]

File: modules/upgrade/pages/migrate/kubernetes/helm-to-operator.adoc [hunk 3]
```diff
@@ -144,75 +144,9 @@ redpanda   True    Redpanda reconciliation succeeded
 
 While the deployment process can sometimes take a few minutes, a prolonged 'not ready' status may indicate an issue.
 
-=== HelmRelease is not ready
+include::manage:kubernetes/troubleshooting/k-troubleshoot.adoc[tags=deployment,leveloffset=-1]
```
Comment (JakeSCahill) [inline modules/upgrade/pages/migrate/kubernetes/helm-to-operator.adoc]:
```suggestion
include::manage:kubernetes/troubleshooting/k-troubleshoot.adoc[tags=deployment]
```

File: modules/upgrade/pages/migrate/kubernetes/helm-to-operator.adoc [hunk 4]
```diff
@@ -235,8 +169,8 @@ kubectl edit pod <operator-name> --namespace <namespace>
 ```bash
 kubectl --namespace <namespace> patch redpanda <cluster-name> -p '{"metadata":{"finalizers":null}}' --type=merge
 kubectl --namespace <namespace> patch helmrelease <cluster-name> -p '{"metadata":{"finalizers":null}}' --type=merge
-kubectl --namespace <namespace> patch helmchart <cluster-name> -p '{"metadata":{"finalizers":null}}' --type=merge
-kubectl --namespace <namespace> patch helmrepository <cluster-name> -p '{"metadata":{"finalizers":null}}' --type=merge
+kubectl --namespace <namespace> patch <namespace>-helmchart <cluster-name> -p '{"metadata":{"finalizers":null}}' --type=merge
+kubectl --namespace <namespace> patch <namespace>-helmrepository <cluster-name> -p '{"metadata":{"finalizers":null}}' --type=merge
 ```
```
Comment (JakeSCahill) [inline modules/upgrade/pages/migrate/kubernetes/helm-to-operator.adoc]:
```suggestion
----
+
Replace the `<helm-repository-name>` placeholder with the value of `spec.chartRef.helmRepositoryName` in your Redpanda resource. The default value is `redpanda-repository`.
```

File: modules/upgrade/pages/migrate/kubernetes/helm-to-operator.adoc [hunk 5]
```diff
@@ -235,8 +169,8 @@ kubectl edit pod <operator-name> --namespace <namespace>
 ```bash
```
Comment (JakeSCahill) [inline modules/upgrade/pages/migrate/kubernetes/helm-to-operator.adoc]:
```suggestion
[,bash]
----
```

File: modules/upgrade/pages/migrate/kubernetes/helm-to-operator.adoc [hunk 6]
```diff
@@ -250,6 +175,18 @@ kubectl delete helmrepository <cluster-name> --namespace <namespace>
...
+
+While the deployment process can sometimes take a few minutes, a prolonged 'not ready' status may indicate an issue.
+
+include::manage:kubernetes/troubleshooting/k-troubleshoot.adoc[tags=deployment]
+
+For more troubleshooting steps, see xref:manage:kubernetes/troubleshooting/k-troubleshoot.adoc[Troubleshoot Redpanda in Kubernetes].
+
+=== Open an issue
+
+If you cannot solve the issue or you need assistance during the migration process, https://github.com/redpanda-data/redpanda/issues/new/choose[open a GitHub issue^] in the Redpanda repository. Before opening a new issue, search the existing issues on GitHub to see if someone has already reported a similar problem or if any relevant discussions that can help you.
```
Comment (RafalKorepta) [inline modules/upgrade/pages/migrate/kubernetes/helm-to-operator.adoc]: I wonder about the place where creating issue is suggested.
```suggestion
If you cannot solve the issue or need assistance during the migration process, https://github.com/redpanda-data/redpanda-operator/issues/new/choose[open a GitHub issue^]. Before opening a new issue, search the existing issues on GitHub to see if someone has already reported a similar problem or if any relevant discussions that can help you.
```

**Review by RafalKorepta — APPROVED** [review]
LGTM no blocking comments.

File: modules/upgrade/pages/migrate/kubernetes/helm-to-operator.adoc [hunk 7]
```diff
@@ -250,6 +175,18 @@ kubectl delete helmrepository <cluster-name> --namespace <namespace>
...
+
+While the deployment process can sometimes take a few minutes, a prolonged 'not ready' status may indicate an issue.
+
+include::manage:kubernetes/troubleshooting/k-troubleshoot.adoc[tags=deployment]
+
+For more troubleshooting steps, see xref:manage:kubernetes/troubleshooting/k-troubleshoot.adoc[Troubleshoot Redpanda in Kubernetes].
+
+=== Open an issue
+
+If you cannot solve the issue or need assistance during the migration process, https://github.com/redpanda-data/redpanda-operator/issues/new/choose[open a GitHub issue^]. Before opening a new issue, search the existing issues on GitHub to see if someone has already reported a similar problem or if any relevant discussions that can help you.
```
Comment (kbatuigas) [inline modules/upgrade/pages/migrate/kubernetes/helm-to-operator.adoc]:
```suggestion
If you cannot solve the issue or need assistance during the migration process, https://github.com/redpanda-data/redpanda-operator/issues/new/choose[open a GitHub issue^]. Before opening a new issue, search the existing issues on GitHub to see if someone has already reported a similar problem or if any relevant discussions can help you.
```
Extra word?

**Review by kbatuigas — CHANGES_REQUESTED** [review]

**Review by kbatuigas — APPROVED** [review]

**Pull request merged**